src/ExtensionComponentWrap/.eslintrc.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
     parserOptions: {
         sourceType: 'module'
+    },
+    rules: {
+        'node/no-unsupported-features': 'off'
```

Comment:
Comment would be helpful to provide context on why we're disabling this

Reply:
This is a temporary fix anyway, since the ExtensionComponentWrap is supposed to move to Peregrine. Added a comment explaining that.